smart_html_client/src/components/PromptInput/PromptInput.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from "react"
+
+import styles from "./PromptInput.module.scss";
+import EnterSvg from "../../assets/enter.svg?react";
+
+interface PromptInputProps {
+    handleEnterDown: (prompt: string) => void;
+    loading: boolean;
+}
+
+const PromptInput: React.FC<PromptInputProps> = ({ handleEnterDown, loading}) => {
```

Comment:
Can the text input area be adjusted in size automatically based on the input content?

Reply:
![image](https://github.com/dyzsasd/SmartHtml/assets/93023129/b49f6530-897f-4073-9e88-b81d6073aa58)
We are using a textarea input field, which supports multi-line input and allows line breaks with Shift+Enter.

Reply:
Oh, find it, nice